### DISCUSSION TITLE
Will Meilisearch work for Hindi Language too?

### DISCUSSION THREAD

**Discussion by aaakashppandey** [meilisearch/product#667] [Feedback & Feature Proposal]
I have integrated Meilisearch with Strapi in my project, I need to search few Hindi Words too, will Meilisearch work for the same.

**Comment by ManyTheFish** [comment]
Hello @aaakashppandey,
First of all, I don't speak Hindi so it's difficult for me to see the Language specificities, don't hesitate to say obvious things that a native speaker would know.

So far, Meilisearch doesn't officially support Hindi, but it doesn't mean that it wouldn't work. If the Hindi Language is mainly space-separated then, in most cases, it would work.
That's been said, if, after trying some query on Meilisearch, you find the results irrelevant or if you can't manage to find the document you are searching for, then send in this discussion some examples of queries, documents, the result you expect and, if you can, why do you expect a specific result.
This would help me to know what should be improved in Meilisearch to officially support the Hindi Language.

Thanks!
